packages/truffle-compile-vyper/index.js

```diff
@@ -0,0 +1,175 @@
+const path = require('path');
+const exec = require('child_process').exec;
+
+const async = require('async');
+const colors = require('colors');
+const minimatch = require('minimatch');
+
+const find_contracts = require('truffle-contract-sources');
+const Profiler = require('truffle-compile/profiler');
+
+const compiler = {
+  name: 'vyper',
+  version: null,
+};
+
+const VYPER_PATTERN = '**/*.vy';
+
+// -------- TODO: Common with truffle-compile --------
+
+const compile = {};
+
+// contracts_directory: String. Directory where .sol files can be found.
+// quiet: Boolean. Suppress output. Defaults to false.
+// strict: Boolean. Return compiler warnings as errors. Defaults to false.
+compile.all = function(options, callback) {
+
+  find_contracts(options.contracts_directory, function(err, files) {
+    if (err) return callback(err)
+
+    options.paths = files;
+    compile.with_dependencies(options, callback);
+  });
+};
+
+// contracts_directory: String. Directory where .sol files can be found.
```

Comment:
These comments might need changing :grin:

Reply:
These comments are part of copy-pasted code from `truffle-compile`, I've left them as is (these will be probably refactored into some common place later).